common/persistence/cassandra/cassandraPersistenceTest.go

```diff
@@ -171,7 +173,7 @@ func (s *TestCluster) LoadVisibilitySchema(fileNames []string, schemaDir string)
 	workflowSchemaDir := path.Join(schemaDir, "visibility")
 	err := loadCassandraSchema(workflowSchemaDir, fileNames, s.cluster.Hosts, s.cluster.Port, s.DatabaseName(), false, nil)
 	if err != nil && !strings.Contains(err.Error(), "AlreadyExists") {
-		log.Fatal(err)
+		s.logger.Fatal("loadCassandraSchema", tag.Error(err))
```

Comment:
[Nit] rename this one to "loadCassandraVisibilitySchema" just so it is clearer which schema failed to actually load